inc/admin/ui/modules/optimization.php

```diff
@@ -233,7 +247,7 @@
 			'type'         => 'textarea',
 			'label_for'    => 'exclude_js',
 			'label_screen' => __( 'JS files to be excludeded from minification and concatenation:', 'rocket' ),
-			'placeholder'  => "e.g. /wp-content/themes/some-theme/(.*).js\ne.g. /wp-content/plugins/some-plugin/script.js",
+			'placeholder'  => "/wp-content/themes/some-theme/(.*).js\n/wp-content/plugins/some-plugin/script.js",
```

Comment:
Done in https://github.com/wp-media/wp-rocket/pull/418/commits/6696b048b0799dbd6882b820ee493ae18814fa38